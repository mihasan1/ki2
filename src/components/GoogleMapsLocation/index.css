.GoogleMapsLocation {
	--shadow-color: #e3e3e3;

	padding-top: 2vh;
}

.GoogleMapsLocation iframe {
	border-radius: 1rem;
	border-top-right-radius: 25%;
	border-bottom-left-radius: 25px;
	box-shadow: 0 0 5px 5px var(--shadow-color);

	height: 75vh;
	width: 80vmin;
}

.dark-mode .GoogleMapsLocation iframe {
	box-shadow: 0px 0px 10px 5px var(--dark-mode-bg-color);
}

@media screen and (max-width: 450px) {
	iframe {
		border-radius: 1rem !important;
	}
}
