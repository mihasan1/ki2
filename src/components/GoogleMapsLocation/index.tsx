import React from "react";
import "./index.css";

const GoogleMapsLocation = () => (
	<div className="GoogleMapsLocation">
		<iframe
			src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d317.5086490084039!2d30.394147178574627!3d50.45843621314652!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40d4cc5fa4d311b3%3A0x8fc5cc5e3d07eab7!2z0JDQndCi0Jog0IbQvC7Qvi7Qui7QsNC90YLQvtC90L7QstCwINCU0L8sINCk0ZbQu9GW0Y8!5e0!3m2!1sru!2sua!4v1582910405937!5m2!1sru!2sua"
			frameBorder="0"
			style={{ border: 0 }}
			allowFullScreen
			title="Розташування Київського авіаційного технікуму"
		/>
	</div>
);

export default GoogleMapsLocation;
