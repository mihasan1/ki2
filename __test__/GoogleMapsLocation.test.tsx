import React from "react";
import renderer from "react-test-renderer";

import GoogleMapsLocation from "./../src/components/GoogleMapsLocation";

test("renders correctly", () => {
	const tree = renderer.create(<GoogleMapsLocation />).toJSON();

	expect(tree).toMatchSnapshot();
});
