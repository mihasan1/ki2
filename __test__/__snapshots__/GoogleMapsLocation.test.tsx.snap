// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="GoogleMapsLocation"
>
  <iframe
    allowFullScreen={true}
    frameBorder="0"
    src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d317.5086490084039!2d30.394147178574627!3d50.45843621314652!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40d4cc5fa4d311b3%3A0x8fc5cc5e3d07eab7!2z0JDQndCi0Jog0IbQvC7Qvi7Qui7QsNC90YLQvtC90L7QstCwINCU0L8sINCk0ZbQu9GW0Y8!5e0!3m2!1sru!2sua!4v1582910405937!5m2!1sru!2sua"
    style={
      Object {
        "border": 0,
      }
    }
    title="Розташування Київського авіаційного технікуму"
  />
</div>
`;
